--- a/configure       2025-05-10 23:16:51.514788300 -0400
+++ b/configure   2025-05-10 23:18:14.621227900 -0400
@@ -16374,7 +16374,7 @@
      extralibs="-lws2_32 -liphlpapi"
      LIBS="$LIBS $extralibs"
      # some winsock2 functions require XP, so WINNT=0x501
-     CFLAGS="$CFLAGS -DWIN32 -D_WIN32_WINNT=0x501"
+     CFLAGS="$CFLAGS -DWIN32 -D_WIN32_WINNT=0x600"
      cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 #include <winsock2.h>
@@ -16435,7 +16435,7 @@


 if test x$is_windows = xyes; then
-  CXXFLAGS="$CXXFLAGS -DWIN32 -D_WIN32_WINNT=0x501"
+  CXXFLAGS="$CXXFLAGS -DWIN32 -D_WIN32_WINNT=0x600"
 fi

 # Note that configure-time endianness detection may be ignored during
